frontend/src/scenes/experimentation/FeatureFlags.js

```diff
@@ -44,6 +45,18 @@ function _FeatureFlags() {
                 return featureFlag.created_by.first_name || featureFlag.created_by.email
             },
         },
+        {
+            title: 'Rollout Precentage',
+            render: function RenderRolloutPrecentage(_, featureFlag) {
+                return `${featureFlag.rollout_percentage}%` || 'Not rolled out'
```

Comment:
If rollout percentage isn't set it doesn't necessarily mean the flag isn't rolled out - it might mean the flag is rolled out based on filters only. 

I think this could actually just be empty and that'd be fine. It would also be consistent with the filters which will be empty if there are no filters.